src/components/Commands.vue

```diff
@@ -253,7 +253,7 @@ export default class Commands extends Vue {
 
         case Command.Decline: {
           ret.push({
-            label: command.data.cost === '~' ? `Decline ${command.data.offer}` : `Decline ${command.data.offer} for ${command.data.cost}`,
+            label: `Decline ${command.data.offer}`,
             command: Command.Decline
```

Comment:
this can be command: `${Command.Decline} ${command.data.offer}`, so logs are more readble

Reply:
Isn't the same result?

Reply:
the log would show `decline 2pw` or `decline up-nav`, instead of just `decline`